lib/order.rb

```diff
@@ -0,0 +1,97 @@
+require_relative 'customer.rb'
+require 'csv'
+
+
+class Order
+  # read-only ID number
+  attr_reader :id, :products, :customer, :fulfillment_status
+
+  
+  def initialize(id, products_hash, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products_hash
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+   
+    # raises ArgumentError if condition isn't met
+    unless [:pending, :paid, :processing, :shipped, :complete].include?(fulfillment_status)
```

Comment:
Nice logic here! I like it! It even makes me feel like the array of valid fulfillment statuses could be pulled into a constant variable